pkgs/top-level/all-packages.nix

```diff
@@ -26475,7 +26475,9 @@ with pkgs;
 
   vivaldi-widevine = callPackage ../applications/networking/browsers/vivaldi/widevine.nix { };
 
-  libopenmpt = callPackage ../applications/audio/libopenmpt { };
+  libopenmpt = callPackage ../applications/audio/libopenmpt {
+    usePulseAudio = config.pulseaudio or stdenv.isLinux;
```

Comment:
Why did you move this to all-packages.nix?

Reply:
hm, I guess `stdenv` is available in there.

1. that's what (some) other packages like `libao` 
2. I thought it was related to deps like this that might only be enabled on certain platforms, and thus that determination might be made at the all-packages level?

I'm not sure, I can change the patch to just be `s/false/stdenv.isLinux`.

Reply:
Most packages I've worked with do these kinds of platform-specific decisions in the package's code instead of the all-packages level. I'd prefer to stick with this & make it `s/false/stdenv.isLinux`.

Reply:
Ack! Done and pushed!